dockerfiles/alpine/apim-acp/README.md

```
@@ -19,27 +19,25 @@ git clone https://github.com/wso2/docker-apim.git
 
 ```
-docker build -t wso2am-acp:4.6.0-alpine .
+docker build -t wso2am-acp:4.7.0-alpine .
 ```
 
 > By default, the Docker image will prepackage the General Availability (GA) release version of the relevant WSO2 product.
 
-> Note:- wso2am-acp:4.6.0-alpine image can only be built on amd64(x86_64). It is not supported to be built or run natively on Apple silicon. But it is possible to build an amd64 image using [Docker buildx](https://docs.docker.com/desktop/multi-arch/) and then run via emulation on rosetta. Use following command.
+> Note:- wso2am-acp:4.7.0-alpine image can only be built on amd64(x86_64). It is not supported to be built or run natively on Apple silicon. But it is possible to build an amd64 image using [Docker buildx](https://docs.docker.com/desktop/multi-arch/) and then run via emulation on rosetta. Use following command.
 
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)